pages/storage-page.ts

```diff
@@ -1,33 +1,47 @@
-import { expect, Locator, Page } from '@playwright/test';
+import { expect, type Locator, type Page } from '@playwright/test';
 
 export class StoragePage {
     readonly page: Page;
     readonly useEncryptionToggle: Locator;
     readonly encryptionEnabled: Locator;
     readonly useLVMToggle: Locator;
     readonly backButton: Locator;
+    readonly actionsAddFileSystem: Locator;
+    readonly addFileSystemButton: Locator;
+    readonly swapPartitionText: Locator;
 
     constructor(page: Page) {
         this.page = page;
         this.useEncryptionToggle = page.getByRole('checkbox', { name: 'Use encryption' });
         this.encryptionEnabled = page.getByRole('button', { name: 'Encryption settings' });
         this.useLVMToggle = page.locator('label').filter({ hasText: /Use logical volume management/ }).locator('span').first();
         this.backButton = page.getByRole('button', { name: 'Back' });
+        this.actionsAddFileSystem = page.getByRole('button', { name: 'Actions' }).first();
+        this.addFileSystemButton = page.getByText('Add file system', { exact: true });
+        this.swapPartitionText = page.getByText('Swap partition');
     }
 
     async useEncryption() {
         await this.useEncryptionToggle.check();
     }
-    
+  
     async validateEncryptionIsUsed() {
         expect(await this.useEncryptionToggle.isChecked()).toBeTruthy();
     }
-    
+
     async useLVM() {
         await this.useLVMToggle.click();
-    }  
+    }
+
+    async accessAddFileSystemPage() {
+        await expect(this.swapPartitionText).toBeVisible({timeout: 2 * 60 * 1000});
```

Comment:
2 minutes to be opened? does it really need so much? could you please provide failure with your investigation on adding to click below some more time, enable state should be automatically detected by Playwright.

Reply:
Removed timeout